google-cloud-core/src/com/google/cloud/tools/intellij/analytics/GoogleUsageTrackerService.java

```diff
@@ -48,7 +48,7 @@
 import org.jetbrains.annotations.Nullable;
 
 /** Google Usage Tracker that reports to Cloud Tools Analytics backend. */
-public class GoogleUsageTracker implements UsageTracker, SendsEvents {
+public class GoogleUsageTrackerService implements UsageTrackerService, SendsEvents {
```

Comment:
Filter out IDEs that you want to completely disable tracking for in here?

Reply:
yeah. I guess I'll go ahead an add AS to keep the behavior the same as before. And we'll revisit this later if it needs to be changed.

Reply:
done: https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/2134/commits/3879306b82dcf3d08ae2daba34fa8af0bea7573c